run-tests.sh

```diff
@@ -2,43 +2,49 @@
 
 set -e
 
-DOCKERFILE_TEMPLATE="tests/Dockerfile.template"
-
 PY_DOCKER_IMAGES=()
-PY_DOCKER_IMAGES+=("2.7.16-slim")
-PY_DOCKER_IMAGES+=("3.7.4-slim")
-PY_DOCKER_IMAGES+=("3.8-slim")
-PY_DOCKER_IMAGES+=("3.9-slim")
-PY_DOCKER_IMAGES+=("3.10-slim")
-PY_DOCKER_IMAGES+=("3.11-slim")
-
-create_docker_file() {
-    local package="$1"
+if [ -n "$TEST_PY_VERSION" ]; then
+	PY_VERSIONS+=("$TEST_PY_VERSION")
+else
+	PY_VERSIONS+=("2.7")
```

Comment:
It still runs on Python 2?

Reply:
The docker tests prove that it does - after all it is just a thin wrapper ;)